.github/workflows/chart_update_on_merge.yml

```diff
@@ -0,0 +1,62 @@
+name: chart-update-on-merge
+
+on:
+  push: # There is no specific "PR merged" event but pushes to protected branch `develop` would only happen during PR merges (and releases) (and Jeff)
+    branches:
+      - develop
+jobs:
+  chart-update:
+    name: Cromwhelm Chart Auto Updater
+    runs-on: self-hosted # Faster machines; see https://github.com/broadinstitute/cromwell/settings/actions/runners
+    steps:
+    - name: Clone Cromwell
+      uses: actions/checkout@v2
+      with:
+        repository: broadinstitute/cromwell
+        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }} # Has to be set at checkout AND later when pushing to work
+        path: cromwell
+    - uses: olafurpg/setup-scala@v10
+      with:
+        java-version: adopt@1.11
+    - name: Clone Cromwhelm
+      uses: actions/checkout@v2
+      with:
+        repository: broadinstitute/cromwhelm
+        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }} # Has to be set at checkout AND later when pushing to work
+        path: cromwhelm
+    - name: Find Cromwell short SHA
+      run: |
+        cd cromwell
+        echo "CROMWELL_SHORT_SHA=`git rev-parse --short $GITHUB_SHA`" >> $GITHUB_ENV
+    - name: Find Cromwell release number
+      run: |
+        previous_version=$(curl -X GET https://api.github.com/repos/broadinstitute/cromwell/releases/latest | jq .tag_name | xargs)
+        echo "CROMWELL_NUMBER=$((previous_version + 1))" >> $GITHUB_ENV
+    - name: Save complete image ID
+      run: |
+        echo "CROMWELL_SNAP_VERSION=`echo "$CROMWELL_NUMBER-$CROMWELL_SHORT_SHA-SNAP"`" >> $GITHUB_ENV
```

Comment:
TOL: In a world where everything is a SNAP, do we still need to say `-SNAP`?

Reply:
Cromwell numbered releases do not really make a splash in the chart other than a higher version number appearing. So yes, they will be called `-SNAP`.

I decided that it is not important to call the release `80-0123456` if an hour later it could be replaced by `80-1234567-SNAP` when someone merges. Therefore, the `-SNAP` is [hard-coded when constructing `CROMWELL_SNAP_VERSION`](https://github.com/broadinstitute/cromwell/pull/6739/files#diff-9df639d711d6c489134fb2df6c9b555e7aa4b472163b925053bae66670c24b25R39).

Basically, my goal was to completely decouple the chart version from the release version. But I am sticking to existing  conventions because changing conventions has a big blast area.